tests/mock_network/utils.rs

```diff
@@ -580,8 +580,124 @@ pub fn sort_nodes_by_distance_to(nodes: &mut [TestNode], name: &XorName) {
     nodes.sort_by(|node0, node1| name.cmp_distance(&node0.name(), &node1.name()));
 }
 
+pub fn verify_individual_chain(node: &TestNode, min_section_size: usize) {
+    let our_prefix = unwrap!(node.inner.our_prefix());
+    let our_name = unwrap!(node.inner.our_name());
+    let our_section_members = node.inner.section_members(our_prefix);
+
+    assert!(
+        our_prefix.matches(our_name),
+        "Our prefix doesn't match our name: {:?}, {:?}",
+        our_prefix,
+        our_name,
+    );
+
+    // TODO (or not?)
+    //assert_eq!(
+    //    chain.prefix_change(),
+    //    PrefixChange::None,
+    //    "{} has an unexpected prefix change: {:?}",
+    //    chain.our_id(),
+    //    chain.prefix_change()
+    //);
```

Comment:
As discussed on Slack, this can probably be safely removed in favour of more explicit checks on lists of members of different sections in our chain - that can be a separate task, though.